src/stacks-hierarchy/Root_TicketInformationScreen.tsx

```diff
@@ -0,0 +1,77 @@
+import {FullScreenView} from '@atb/components/screen-view';
+import {ThemeText} from '@atb/components/text';
+import {RootStackScreenProps} from '@atb/stacks-hierarchy/navigation-types';
+import {
+  getTextForLanguage,
+  PurchaseOverviewTexts,
+  useTranslation,
+} from '@atb/translations';
+import {ScrollView, View} from 'react-native';
+import {StyleSheet} from '@atb/theme';
+import {GenericSectionItem, Section} from '@atb/components/sections';
+import {SectionHeading} from '@atb/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_RootScreen/components/SectionHeading';
+import {TransportationIconBoxList} from '@atb/components/icon-box';
+
+type Props = RootStackScreenProps<'Root_TicketInformationScreen'>;
+
+export const Root_TicketInformationScreen = (props: Props) => {
+  const {t, language} = useTranslation();
+  const styles = useStyle();
+  const fareProductTypeConfig = props.route.params.fareProductTypeConfig;
+  const preassignedFareProduct = props.route.params.preassignedFareProduct;
```

Comment:
Maybe just pass the id of the `preassignedFareProduct` here, and retrieve the `preassignedFareProduct` and `fareProductTypeConfig` objects with it. That is more inline with the [React Navigation doc](https://reactnavigation.org/docs/params/#what-should-be-in-params):
> Params are like options for a screen. They should only contain information to configure what's displayed in the screen. Avoid passing the full data which will be displayed on the screen itself (e.g. pass a user id instead of user object)

Reply:
I agree that passing id's is cleaner, but I thought passing objects was fine as long as it's serialised in a predictable way?

Reply:
I get the point after reading the docs more thoroughly. Thank you!